### PR TITLE
feat: Add more constants core-data and metadatai

### PR DESCRIPTION
**Pull request by cloudxxx8**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## What is the current behavior?

V2 API implementation needs the following constants:
- deviceName
- created
- modified
- valueType
- serviceName
- profileName

## Issue Number:
Fix https://github.com/edgexfoundry/go-mod-core-contracts/issues/347

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports
Are there any new imports or modules? If so, what are they used for and why?

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

**Review by judehung — COMMENTED** [review]
The issue mentioned that we need valueType, but I don't valueType added as one of the constants.  Do you miss it?

**Comment by cloudxxx8** [comment]
> The issue mentioned that we need valueType, but I don't valueType added as one of the constants. Do you miss it?

right, I missed it.  added it.  thanks.

File: v2/constants.go [hunk 1]
```diff
@@ -51,23 +51,28 @@ const (
...
+	End          = "end"
+	Age          = "age"
+	Scrub        = "scrub"
+	Type         = "type"
+	Name         = "name"
+	Manufacturer = "manufacturer"
+	Model        = "model"
+	Offset       = "offset" //query string to specify the number of items to skip before starting to collect the result set.
+	Limit        = "limit"  //query string to specify the numbers of items to return
+	Labels       = "labels" //query string to specify associated user-defined labels for querying a given object. More than one label may be specified via a comma-delimited list
```
Comment (judehung) [inline v2/constants.go:75]: I still couldn't find `ValueType` as defined in the constant list.  Do you wanna check one more time?

Comment (cloudxxx8) [reply v2/constants.go]: sorry, I missed a git step.  it is good now

**Review by judehung — APPROVED** [review]
LGTM

**Pull request merged**